src/app/content.rs

```diff
@@ -72,11 +75,24 @@ impl SimpleComponent for ContentModel {
                         None => adw::StatusPage {
                             set_title: "No Pet Selected",
                         }
-                        Some(pet) => &gtk::Label {
-                            #[watch] set_label: &pet.name,
-                            set_margin_all: 4,
-                            set_css_classes: &["title-1"],
-                            set_vexpand: true,
+                        Some(_) => &adw::Clamp {
+                            set_margin_all: 16,
+
+                            gtk::Box {
+                                set_orientation: gtk::Orientation::Vertical,
+                                set_spacing: 16,
+
+                                gtk::StackSwitcher {
+                                    set_stack: Some(&stack)
+                                },
+
+                                #[name(stack)]
+                                gtk::Stack {
+                                    set_transition_type: gtk::StackTransitionType::Crossfade,
+
+                                    add_named[Some("Info")] = model.info_view.widget(),
+                                }
```

Comment:
I just noticed it now.

A stack with only one page?
An a switcher?

I did see it in early rebasings and fixed it.
I must have cherry-picked later and missed this.